docs/en/guides/pipeline-integrations/getting-started/dynamic_filesystem_configuration.md

```diff
@@ -128,7 +128,7 @@ You’ve now enabled a Set custom entity, created a Set entity called “Dining
 
 Now it’s time to define the folder structure you want Toolkit to dynamically generate as artists step through the production pipeline. This is done by editing the schema.
 
-{% include info title="Note" content="It’s a good practice to test configurations in a *cloned* configuration, so as not to affect the active production configuration. The cloning process creates a *copy* of the configuration that you can edit safely before pushing changes to the live config. You can find details on cloning your config in the [Configuration Staging and Rollout](https://support.shotgunsoftware.com/hc/en-us/articles/219033168-Configuration-staging-and-rollout#Cloning%20your%20Configuration) document." %}
+{% include info title="Note" content="It’s a good practice to test configurations in a *cloned* configuration, so as not to affect the active production configuration. The cloning process creates a *copy* of the configuration that you can edit safely before pushing changes to the live config. You can find details on cloning your config in the [Configuration Staging and Rollout](https://developer.shotgridsoftware.com/60762324/?title=Configuration+Staging+and+Rollout#cloning-your-configuration) document." %}
```

Comment:
curly quote

Reply:
Thanks! Cleaned up a few of these I noticed lingering around.